### DISCUSSION TITLE
Recent issue where icon, but not video stream shows on dashboard.

### DISCUSSION THREAD

**Discussion by fractal815** [Motion-Project/motion#1830] [Q&A]
kUbuntu 24.04, MOTION 4.6.0 Dedicate machine.   I have been using motion for several years , and have not updated any configuration in a long time. Recently, Icons are displaying on the dashboard, but not video streams.   Clicking on the icon produces a web server area.    I have separate conf files for each of my camera threads.   Within each of these conf files I set a specific "stream_port" value for the particular camera.    Withing the motion.conf the default value is still set as "stream_port 8081".    This configuration has worked for years.   The system otherwise still works fine, with recordings being produced when motion is detected.       Today I took some time to investigate, and commented out the default "stream_port 8081" value in the motion.conf file, and this then corrected the problem.    It appears that the program is no longer superseding the motion.conf default value with the value in any separate thread config file.

**Comment by Mr-Dave** [comment]
The webcontrol page will use the `stream_port` specified in the motion.conf file for the web page if it is specified.  If there is a different port specified in the camera.conf file, it will also provide a stream on that port.  If you want to have the webcontrol page use the unique port numbers, then specify the `stream_port` in the motion.conf file as zero.  Note that this is intentional and not a bug.

**Reply by fractal815** [reply]
Thanks for your quick reply.   As I have only recently seen this behavior I am wondering if this a recent change, or if I've just been "lucky" as I have only seen it recently.   Regardless, I do have it corrected now and my post was more about giving visibility to the issue.

**Reply by Mr-Dave** [reply]
To be clear.  The ability to specify a single port or multiple ports for the camera streams has been around for quite a few years.  (This is the recognition of the stream_port in the motion.conf file vs the ones in the camera.conf files).  

I'd also note that I get images on the webcontrol either way.  That is, whether the stream_port is specified the motion.conf or differently in the camera.conf.  The only difference is which one is being used by the webcontrol page.
